src/main/java/org/dependencytrack/resources/v1/vo/BomSubmitRequest.java

```diff
@@ -72,29 +72,31 @@ public final class BomSubmitRequest {
 
     private final boolean autoCreate;
 
-    private final boolean isLatestProjectVersion;
+    private final Boolean isLatest;
 
     public BomSubmitRequest(String project,
                             String projectName,
                             String projectVersion,
                             List<Tag> projectTags,
                             boolean autoCreate,
-                            boolean isLatestProjectVersion,
+                            boolean isLatest,
                             String bom) {
-        this(project, projectName, projectVersion, projectTags, autoCreate, null, null, null, isLatestProjectVersion, bom);
+        this(project, projectName, projectVersion, projectTags, autoCreate, null, null, null, isLatest, isLatest, bom);
     }
 
     @JsonCreator
-    public BomSubmitRequest(@JsonProperty(value = "project") String project,
-                            @JsonProperty(value = "projectName") String projectName,
-                            @JsonProperty(value = "projectVersion") String projectVersion,
-                            @JsonProperty(value = "projectTags") List<Tag> projectTags,
-                            @JsonProperty(value = "autoCreate") boolean autoCreate,
-                            @JsonProperty(value = "parentUUID") String parentUUID,
-                            @JsonProperty(value = "parentName") String parentName,
-                            @JsonProperty(value = "parentVersion") String parentVersion,
-                            @JsonProperty(value = "isLatestProjectVersion", defaultValue = "false") boolean isLatestProjectVersion,
-                            @JsonProperty(value = "bom", required = true) String bom) {
+    public BomSubmitRequest(
+            @JsonProperty(value = "project") String project,
+            @JsonProperty(value = "projectName") String projectName,
+            @JsonProperty(value = "projectVersion") String projectVersion,
+            @JsonProperty(value = "projectTags") List<Tag> projectTags,
+            @JsonProperty(value = "autoCreate") boolean autoCreate,
+            @JsonProperty(value = "parentUUID") String parentUUID,
+            @JsonProperty(value = "parentName") String parentName,
+            @JsonProperty(value = "parentVersion") String parentVersion,
+            @JsonProperty(value = "isLatestProjectVersion", defaultValue = "false") boolean isLatestProjectVersion,
+            @JsonProperty(value = "isLatest") Boolean isLatest,
```

Comment:
Consider using [`@JsonAlias`](https://www.baeldung.com/jackson-annotations#6-jsonalias) instead of defining parameters for both variants.

Reply:
Done